docs/guide/config_repository.md

```diff
@@ -89,6 +101,18 @@ ezplatform:
             repository: second_repository
 ```
 
+```
+# .env.local
+DATABASE_USER=ezplatform
+DATABASE_PASSWORD=password
+DATABASE_NAME=ezplatform
+DATABASE_HOST=localhost
+DATABASE_PORT=3306
+DATABASE_PLATFORM=mysql
```

Comment:
@vidarl do we really need all those variables? It's done like that for Docker setup, but I don't think it applies to a second instance as well.

Reply:
You are right. In this case I could use the actual values directly when setting `SECOND_DATABASE_URL`

Reply:
Sorry @alongosz, forgot about your suggestion.. Looks OK now ?

Reply:
Guys, sorry to interfere, but based on this PR:
https://github.com/ezsystems/ezplatform/pull/576
should you include the DATABASE_VERSION parameter here as well?

Reply:
Ok, I was not aware of that [fresh change](https://github.com/ezsystems/ezplatform/pull/576)...
I have added DATABASE_VERSION now